obstacle_range: 2.5
raytrace_range: 3.0
footprint: [[0.2, 0.2], [0.2, -0.2], [-0.2, -0.2], [-0.2, 0.2]]
robot_radius: 0.3
inflation_radius: 0.55
transform_tolerance: 0.5
